<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>测试报告</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.1"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 80%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
4:Error  
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2 || level ==4) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('case_detail');
    //console.log(detail_class.length)
    if (level == 3) {
        for (var i = 0; i < detail_class.length; i++){
            detail_class[i].innerHTML="收起"
        }
    }
    else{
            for (var i = 0; i < detail_class.length; i++){
            detail_class[i].innerHTML="详细"
        }
    }
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h4 style="font-family: Microsoft YaHei">测试报告</h4>
<p class='attribute'><strong>测试人员 : </strong> tester1,tester2</p>
<p class='attribute'><strong>开始时间 : </strong> 2021-01-27 20:10:37</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:12:49.292308</p>
<p class='attribute'><strong>测试结果 : </strong> 共 20，通过 5，失败 1，错误 14，通过率= 25.00%</p>

<p class='description'>执行情况：</p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 25.00% }</a>
<a class="btn btn-warning" href='javascript:showCase(4)'>错误{ 14 }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 1 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 5 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 20 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 14px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
    <td>截图</td>
</tr>

<tr class='errorClass warning'>
    <td>dynamicModule.AndroidTests</td>
    <td class="text-center">4</td>
    <td class="text-center">1</td>
    <td class="text-center">0</td>
    <td class="text-center">3</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',4)" class="case_detail" id='c1'>详细</a></td>
    <td class="text-center">Assert or Error Image</td>
</tr>

<tr id='pt1_1' class='hiddenRow'>
    <td class='passCase' width='300px'><div class='testcase'>test_dynamicModule01: 验证动态列表显示情况</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_1'>通过</button>
    <div id='div_pt1_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_1'>通过</button>
    <div id='div_pt1_1' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
pt1_1: 关注
关注


    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft1_2' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>test_dynamicModule02: 验证评论动态功能是否正常</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft1_2: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\dynamicModule.py", line 46, in test_dynamicModule02
    self.driver.find_element_by_id("cn.xiaochuankeji.tieba:id/tvCommentCount").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: read ECONNRESET



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft1_3' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>test_dynamicModule03: 验证评论动态功能是否正常</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft1_3: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\dynamicModule.py", line 57, in test_dynamicModule03
    self.driver.find_element_by_id("cn.xiaochuankeji.tieba:id/tvCommentCount").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: read ECONNRESET



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft1_4' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>test_dynamicModule04: 验证点击头像进入用户主页进行私信情况</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft1_4: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\dynamicModule.py", line 68, in test_dynamicModule04
    self.driver.find_element_by_xpath("//android.view.View[@resource-id='cn.xiaochuankeji.tieba:id/avatar_view_root']/android.widget.ImageView[1]").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: read ECONNRESET



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr class='errorClass warning'>
    <td>myModule.AndroidTests</td>
    <td class="text-center">1</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">1</td>
    <td class="text-center"><a href="javascript:showClassDetail('c2',1)" class="case_detail" id='c2'>详细</a></td>
    <td class="text-center">Assert or Error Image</td>
</tr>

<tr id='ft2_1' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>test_myModule01: 验证“我的”页面展示情况</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft2_1: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\myModule.py", line 27, in setUp
    self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 155, in __init__
    proxy
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 157, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 225, in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, parameters)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: pkg: /data/local/tmp/UnicodeIME-debug.apk
Failure [INSTALL_FAILED_INVALID_URI]



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr class='errorClass warning'>
    <td>news_modules.AndroidTests</td>
    <td class="text-center">2</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">2</td>
    <td class="text-center"><a href="javascript:showClassDetail('c3',2)" class="case_detail" id='c3'>详细</a></td>
    <td class="text-center">Assert or Error Image</td>
</tr>

<tr id='ft3_1' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>test_newsModule01: 验证消息页面显示是否正常</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_1'>错误</button>
    <div id='div_ft3_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_1'>错误</button>
    <div id='div_ft3_1' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft3_1: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\news_modules.py", line 27, in setUp
    self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 155, in __init__
    proxy
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 157, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 225, in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, parameters)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: read ECONNRESET



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft3_2' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>test_newsModule02: 验证回复私信功能是否正常</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft3_2: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\news_modules.py", line 44, in test_newsModule02
    self.driver.find_element_by_xpath("//android.widget.TextView[@text='消息']").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: android.support.test.uiautomator.StaleObjectException
	at android.support.test.uiautomator.UiObject2.getAccessibilityNodeInfo(UiObject2.java:629)
	at android.support.test.uiautomator.UiObject2.getVisibleBounds(UiObject2.java:210)
	at android.support.test.uiautomator.UiObject2.getVisibleCenter(UiObject2.java:251)
	at android.support.test.uiautomator.UiObject2.click(UiObject2.java:343)
	at io.appium.uiautomator2.model.UiObject2Element.click(UiObject2Element.java:65)
	at io.appium.uiautomator2.handler.Click.safeHandle(Click.java:38)
	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:37)
	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:252)
	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:243)
	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:44)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:514)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:468)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
	at java.lang.Thread.run(Thread.java:818)




    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr class='errorClass warning'>
    <td>shouyeApp.AndroidTests</td>
    <td class="text-center">12</td>
    <td class="text-center">4</td>
    <td class="text-center">1</td>
    <td class="text-center">7</td>
    <td class="text-center"><a href="javascript:showClassDetail('c4',12)" class="case_detail" id='c4'>详细</a></td>
    <td class="text-center">Assert or Error Image</td>
</tr>

<tr id='ft4_1' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>testshouye01: 验证关注功能</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_1'>错误</button>
    <div id='div_ft4_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_1'>错误</button>
    <div id='div_ft4_1' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_1: Traceback (most recent call last):
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1252, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1298, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1247, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1065, in _send_output
    self.send(chunk)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 987, in send
    self.sock.sendall(data)
ConnectionAbortedError: [WinError 10053] 你的主机中的软件中止了一个已建立的连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 82, in testshouye01
    self.driver.find_element_by_xpath("//android.widget.TextView[@text='关注']").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 319, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 374, in execute
    return self._request(command_info[0], url, body=data)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 397, in _request
    resp = self._conn.request(method, url, body=body, headers=headers)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\util\retry.py", line 400, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\packages\six.py", line 734, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\urllib3\connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1252, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1298, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1247, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1065, in _send_output
    self.send(chunk)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 987, in send
    self.sock.sendall(data)
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None))


    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft4_2' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>testshouye02: 验证关注板块是否支持滑动</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_2'>错误</button>
    <div id='div_ft4_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_2'>错误</button>
    <div id='div_ft4_2' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_2: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 99, in testshouye02
    self.driver.find_element_by_xpath("//android.widget.TextView[@text='关注']").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft4_3' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>testshouye03</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_3'>错误</button>
    <div id='div_ft4_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_3'>错误</button>
    <div id='div_ft4_3' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_3: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 109, in testshouye03
    rs = self.driver.find_element_by_id("cn.xiaochuankeji.tieba:id/expand_content_view")
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft4_4' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>testshouye04: 验证用户头像跳转情况</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_4'>错误</button>
    <div id='div_ft4_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_4'>错误</button>
    <div id='div_ft4_4' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_4: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 124, in testshouye04
    btn_home = self.driver.find_element_by_id("cn.xiaochuankeji.tieba:id/btn_profile")
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: read ECONNRESET



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft4_5' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>testshouye05: 验证详情页返回首页</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_5'>错误</button>
    <div id='div_ft4_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_5'>错误</button>
    <div id='div_ft4_5' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_5: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 140, in testshouye05
    self.assertEqual(rs_nav[3].text, "图文")
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 76, in text
    return self._execute(Command.GET_ELEMENT_TEXT)['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy. Proxy error: Could not proxy command to remote server. Original error: Error: socket hang up



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='ft4_6' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>testshouye06: 验证是否可以评论帖子</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_6'>错误</button>
    <div id='div_ft4_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_6'>错误</button>
    <div id='div_ft4_6' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_6: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 147, in testshouye06
    self.driver.find_element_by_id("cn.xiaochuankeji.tieba:id/etInput").send_keys("哈哈哈")
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: connect ECONNREFUSED 127.0.0.1:8200



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='pt4_7' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testshouye07: 验证评论帖子页面返回</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200" width="400"/>
        </a>
    </td>
</tr>

<tr id='ft4_8' class='none'>
    <td class='failCase' width='300px'><div class='testcase'>testshouye08: 验证搜索功能是否正常</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_8'>失败</button>
    <div id='div_ft4_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_8'>失败</button>
    <div id='div_ft4_8' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_8: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 172, in testshouye08
    self.assertIn("新闻",rs)
AssertionError: '新闻' not found in 'sdfdf'


    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='pt4_9' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testshouye09: 验证取消搜索情况</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200" width="400"/>
        </a>
    </td>
</tr>

<tr id='pt4_10' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testshouye10: 验证清空“搜索输入”情况</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200" width="400"/>
        </a>
    </td>
</tr>

<tr id='pt4_11' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testshouye11: 验证转发取消功能是否正常</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200" width="400"/>
        </a>
    </td>
</tr>

<tr id='ft4_12' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>testshouye12: 验证不喜欢功能是否正常</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_12'>错误</button>
    <div id='div_ft4_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_12'>错误</button>
    <div id='div_ft4_12' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft4_12: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\shouyeApp.py", line 206, in testshouye12
    self.driver.find_element_by_xpath("//android.widget.TextView[@text='视频']").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr class='errorClass warning'>
    <td>topicModule.AndroidTests</td>
    <td class="text-center">1</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">1</td>
    <td class="text-center"><a href="javascript:showClassDetail('c5',1)" class="case_detail" id='c5'>详细</a></td>
    <td class="text-center">Assert or Error Image</td>
</tr>

<tr id='ft5_1' class='none'>
    <td class='errorCase' width='300px'><div class='testcase'>test_topicModule02: 验证发布功能是否正常</div></td>
    <td colspan='5' align='left' width='600px'> <!--print 输出框位置-->
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse in">
    <pre style="overflow-y:scroll; overflow-x:hidden;height:200px; width:600px; margin:auto; border:1px solid #e1e1e1;">
    
ft5_1: Traceback (most recent call last):
  File "D:\编测编学\selenium\day03\Uiframe0test\case\app\topicModule.py", line 41, in test_topicModule02
    self.driver.find_element_by_xpath("//android.widget.TextView[@text='视频']").click()
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\webdriver.py", line 281, in find_element
    'value': value})['value']
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Users\JHQ\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.



    </pre>
    </div>
    </td>
    <td align="right">
        <a hidden="hidden" href="../images/">
            <img   src="../images/" height="200px" width="400px"/>
        </a>
    </td>
</tr>



<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>20</td>
    <td>5</td>
    <td>1</td>
    <td>14</td>
    <td>通过率：25.00%</td>
    <td> <a href="" target="_blank"></a></td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
